data/remnant/remnant 1 introduction.txt

```diff
@@ -3741,4 +3742,73 @@ mission "Remnant: Expanded Horizons Storms 1"
+					goto delivery
+			label houses
+			`	"You do not?" he responds playfully. "We maintain a decentralized power grid so we have more resiliency in the event of an attack or natural disaster. It also makes use of the power plants we salvage from the Korath and ones we recover from our own damaged ships. Besides, widely deploying the technology ensures that we have expertise to maintain and repair it all across the Remnant. But," he signs a pushing motion, "civil engineering is another subject. I hope you are more interested in how we move our various cargoes."`
+				goto delivery
+			label delivery
+			`	You nod to agree and he continues, "Those reactors were originally scheduled for delivery 30 days from now, but they were ready earlier and the project is able the install them now. So, by bringing them over, you have enabled us to move the project forward and free up delivery capacity next month."`
+			`	"So, why did you offer this work to me?" you ask, trying to mimic the gestures you've seen that you think refer to one's self.`
+			`	He smiles and responds, "Very good! Very good! We schedule everything, but we maintain slack in the system to take advantage of opportunities like the reactors being ready early, or to deal with emergencies. Sometimes, of course, we have to drop everything and sort the schedule out later. But, in normal times, we publish these jobs on the job board and captains that have capacity available can pick them up. The projects provide resource allocations to cover operating and crew costs, but there is no...what is the word? Margin, yes! Since your ship is not in our planned resources, you provide extra capacity, and we might be able to move some of our larger projects forward when you are in our space.`
+			choice
+				`	"Will your pilots be upset at the competition?"`
+					goto competition
+				`	"Thanks for the opportunity."`
+					goto thanks
+			label competition
+			`	Eon smiles at the question. "You should talk to the other pilots and see, but I think the ancestral approach is different. We work more like a military: the ships are owned by the Remnant, but each captain is responsible for the ship, the crew, and the transport work that needs to be done. I think they will be happy to see new homes being built for   families, and," -he winks at you- "spending more time with their own."`
+				goto thanks
+			label thanks
+			`	Before you can ask more about transport jobs, Eon takes out his commlink and taps on it for a few minutes. "You now have access to the logistics job board. You should be able to see our current job list for each system when you check."`
+			`	You chat with Eon about how you've outfitted the <ship> as you finish your...coffee. Then, his commlink chirps. As he stands, he signs what you think means "goodbye" as he says he has to leave for another meeting. He walks off quickly with his head down looking at the datapad. Your commlink notifies you that <payment> have been added to your account.`
```

Comment:
```suggestion
			`	You continue to chat with Eon as you drink your coffee. When you finish, his commlink chirps. As he stands, he signs what you think means "goodbye" as he says he has to leave for another meeting. He walks off quickly with his head down looking at the datapad. Your commlink notifies you that <payment> have been added to your account.`
```

Reply:
Personal preference: I dislike coffee, and won't even contemplate my characters drinking it. So I'd prefer to read about my pilot drinking their beverage; but that's fairly minor.